### ISSUE TITLE
Version 2.5.4 is breaking

### ISSUE THREAD

**Issue by ShlomiAltostra** [ReactiveX/IxJS#299]
**IxJS version: [2.5.4](https://www.npmjs.com/package/ix/v/2.5.4)**

**Code to reproduce:**

```js
import { concat } from '@reactivex/ix-es2015-cjs/iterable/pipe/concat' // Cannot find module
```

**Expected behavior:**
Importing specific pipe operator from `iterable/pipe/*` should work

**Actual behavior:**
Error of cannot find module

**Comment by trxcllnt** [comment]
Yeah sorry had a publish snafu saturday and accidentally published 3.0.1 with the 2.5.4 tag. I'll re-publish 2.5.3 as 2.5.5 today.

**Comment by trxcllnt** [comment]
Just re-published the v2.5.3 tag as v2.5.5, let me know if that doesn't fix it. Sorry for the confusion!

**Comment by ShlomiAltostra** [comment]
Thanks!
